Sources/FluentUI_iOS/Core/Extensions/Color+Extensions.swift

```diff
@@ -14,7 +14,7 @@ extension Color {
     /// - Parameter hexValue: The color value to store, in 24-bit (three-channel, 8-bit) RGB.
     ///
     /// - Returns: A color object that stores the provided color information.
-    init(hexValue: UInt32) {
+	public init(hexValue: UInt32) {
```

Comment:
nit: spacing

Reply:
also I only see the other init being used, any reason to expose this init or just for future use?